.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - '6'
```

Comment:
this should probably be considered a breaking change and will require a new major version release